src/vm/hints/usort.rs

```diff
@@ -0,0 +1,227 @@
+use crate::{
+    bigint, bigintusize,
+    serde::deserialize_program::ApTracking,
+    types::exec_scope::PyValueType,
+    vm::{
+        errors::vm_errors::VirtualMachineError,
+        hints::hint_utils::{
+            get_int_from_scope, get_integer_from_relocatable_plus_offset,
+            get_integer_from_var_name, get_key_to_list_map_from_scope_mut, get_list_from_scope_mut,
+            get_list_from_scope_ref, get_range_check_builtin, get_relocatable_from_var_name,
+            insert_integer_at_relocatable_plus_offset, insert_integer_from_var_name,
+            insert_relocatable_from_var_name,
+        },
+        vm_core::VirtualMachine,
+    },
+};
+use num_bigint::BigInt;
+use num_traits::{FromPrimitive, ToPrimitive};
+use std::collections::HashMap;
+
+pub fn usort_enter_scope(vm: &mut VirtualMachine) -> Result<(), VirtualMachineError> {
+    let usort_max_size =
+        get_int_from_scope(vm, "usort_max_size").map_or(PyValueType::None, PyValueType::BigInt);
+    vm.exec_scopes.enter_scope(HashMap::from([(
+        "usort_max_size".to_string(),
+        usort_max_size,
+    )]));
+    Ok(())
+}
+
+pub fn usort_body(
+    vm: &mut VirtualMachine,
+    ids: &HashMap<String, BigInt>,
+    hint_ap_tracking: Option<&ApTracking>,
+) -> Result<(), VirtualMachineError> {
+    let _ = get_range_check_builtin(vm)?;
```

Comment:
Why are we searching this and then ignoring the result?

Reply:
resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/319/commits/77802f6aed25749adf837d8efcb417b01b735d74